packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.js

```diff
@@ -360,6 +359,26 @@ export default class InlineContent extends Base {
         text: text.text || text
       }
     };
+    if (textOverrideStyles?.textDecoration === 'line-through') {
+      const textWidth = measureTextWidth({
+        ...this.style.textStyle,
+        ...textOverrideStyles,
+        text: text.text || text
+      });
+      const strikethroughLine = {
+        rect: true,
+        w: textWidth - 2,
+        h: 2,
```

Comment:
the `h` might need to be dynamic (a percentage of the text component's h). for example what if the font size is 40? the strikethrough might appear too thin.

Reply:
Great point